src/Microsoft.Health.Dicom.Functions/Configs/SqlServerConfiguration.cs

```diff
@@ -5,19 +5,19 @@
 
 using System.ComponentModel.DataAnnotations;
 
-namespace Microsoft.Health.Dicom.Functions.Indexing.Configuration
+namespace Microsoft.Health.Dicom.Functions.Configs
 {
     /// <summary>
     /// Represents configuration settings related to the indexing of data.
     /// </summary>
-    public class IndexingConfiguration
+    public class SqlServerConfiguration
```

Comment:
remove this

Reply:
+1, we should not have store specific things in generic projects.

Why do you need connectionString?

Reply:
If you need SQL, you should be able to reference DICOM.core

Reply:
Azure Function is running on different host than Dicom service, so not able to access the [config](https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Web/appsettings.json) file , instead it should read config from [host.json](https://github.com/microsoft/dicom-server/blob/features/ExtendedQueryTagsV2/src/Microsoft.Health.Dicom.Functions/host.json), the same connection string will be duplicated to host.json and consumed by Azure Function.